app/main/views/briefs.py

```diff
@@ -183,12 +183,21 @@ def view_response_result(brief_id):
 
 
 def _render_not_eligible_for_brief_error_page(brief, clarification_question=False):
-    sf = get_supplier_framework_info(data_api_client, brief['frameworkSlug'])
-    on_framework = sf.get('onFramework', False) if sf else False
+    # let's poke the api to discover whether we have any services when queried with less restrictive terms
+    common_kwargs = {
+        "supplier_id": current_user.supplier_id,
+        "framework": brief['frameworkSlug'],
+    }
+    has_framework_service = bool(data_api_client.find_services(**common_kwargs)["services"])
```

Comment:
This probably is slightly more precise than checking `on_framework` (to catch the case of a supplier having all their services disabled), but the result will be exactly the same in 100% of cases as we haven't ever disabled a DOS service as far as I know.